### PR TITLE
platforms/azure: swap default instance size

### PR DESCRIPTION
**Pull request by robszumski**
Upstreaming https://github.com/robszumski/tectonic-installer/pull/1/files

### CODE REVIEW

File: platforms/azure/terraform.tfvars.example [hunk 1]
```diff
@@ -20,9 +20,9 @@ tectonic_cl_channel = "stable" # alpha, beta, stable
 
 # Azure Details
 tectonic_azure_ssh_key = "/Users/coreos/.ssh/id_rsa.pub"
-tectonic_azure_etcd_vm_size = "Standard_DS2_v2"
-tectonic_azure_master_vm_size = "Standard_DS2_v2"
-tectonic_azure_worker_vm_size = "Standard_DS2_v2"
+tectonic_azure_etcd_vm_size = "Standard_DS2_v2_v2"
+tectonic_azure_master_vm_size = "Standard_DS2_v2_v2"
+tectonic_azure_worker_vm_size = "Standard_DS2_v2_v2"
```
Comment (cehoffman) [inline platforms/azure/terraform.tfvars.example]: Should be `Standard_DS2_v2`, there is no `Standard_DS2_v2_v2`

Comment (robszumski) [inline platforms/azure/terraform.tfvars.example]: Heh, whoops, find and replace gone amuck

**Comment by philips** [comment]
Re-running the tests.

**Comment by robszumski** [comment]
Fixed conflicts

**Pull request merged**